src/autopas/tuning/AutoTuner.cpp

```diff
@@ -290,7 +307,13 @@ void AutoTuner::bumpIterationCounters() {
 bool AutoTuner::willRebuildNeighborLists() const {
   const bool inTuningPhase = this->inTuningPhase();
   // How many iterations ago did the rhythm of rebuilds change?
-  const auto iterationBaseline = inTuningPhase ? (_iterationsSinceTuning - _tuningInterval) : _iterationsSinceTuning;
+  // If we are in a tuning phase and in the first tuning iteration but tuneConfiguration has not been called yet
+  // (_iterationsSinceTuning == _tuningInterval) we have 0 as baseline, otherwise the baseline are the iterations we
+  // already did in this tuning phase. If we are not in a tuning phase we subtract the the number of iterations from the
+  // previous tuning phase to get the baseline
```

Comment:
```suggestion
  // If we are in a tuning phase and in the first tuning iteration but tuneConfiguration has not been called yet
  // (_iterationsSinceTuning == _tuningInterval) we have 0 as the baseline, otherwise, the baseline is the iterations we
  // already did in this tuning phase. If we are not in a tuning phase, we subtract the number of iterations from the
  // previous tuning phase to get the baseline
```

Reply:
done in [b9e4111](https://github.com/AutoPas/AutoPas/pull/943/commits/b9e4111c93ce42af332b045755214316b102b040)